src/ProgressOnderwijsUtils/Data/QueryTableValuedParameterComponent.cs

```diff
@@ -63,6 +63,31 @@ public void ToSqlParameter(ref SqlParamArgs paramArgs)
     }
 }
 
+sealed class TvpEqualityToken<TIn>(IEnumerable<TIn> Values, string DbTypeName) : IEquatable<TvpEqualityToken<TIn>>
```

Comment:
This is the only actually "significant" change; and its covered by tests here: https://github.com/progressonderwijs/ProgressOnderwijsUtils/pull/1088/files#diff-be8ab802291dc534840fba104ab0675ae7a03d5aa0dda37d24e50a791c4cc067R119